llvm/test/Transforms/GVNSink/pr138345.ll

```diff
@@ -0,0 +1,51 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; REQUIRES: x86-registered-target
+; RUN:  opt -passes="gvn-sink" -S %s | FileCheck %s
+; RUN:  opt -passes="gvn-sink,correlated-propagation" -S %s | FileCheck %s
```

Comment:
No need to run cvp here, it's not related. You only get a different error because you're passing invalid IR to it.

Reply:
done.